### ISSUE TITLE
The model cannot converge

### ISSUE THREAD

**Issue by DruncBread** [tlc121/FsFont#20]
I tried small epoch(1500) to see the performance. but the loss jumps around 0.04~0.05 and i didnt see it get smaller, is this normal? 
heres my dataset and hyperparams, any advice?
how was ur training process?  is it okay to share ur logs maybe?
![6ab1a59fc87e0de21dbb6a21214f988](https://user-images.githubusercontent.com/100769927/224064094-339d9f2b-ba6a-4670-985c-d23fd3066e65.png)
![Snipaste_2023-03-09_22-52-09](https://user-images.githubusercontent.com/100769927/224064121-48b427b3-12bd-43d7-bc5e-ca09bd52c916.png)
